### PR TITLE
BAM-2006 add feeds section to docs

### PR DESCRIPTION
**Pull request by xak200**
used:
https://docs.google.com/document/d/1qiAltDQbkHkv86TXAQcSN7-svwN5c8jSe98KMlwPJKQ/edit

https://bamerati.atlassian.net/wiki/spaces/BI/pages/491552/How+to+Upload+Product+Feed+Files+to+BAM+via+FTP+and+SFTP

https://docs.google.com/document/d/17OmANp9HeCOgAP-QOpBhuWNJFYXMmS5OEhvHDFp5DYg/edit

screen shot 2018-12-11 at 4 44 46 pm

screen shot 2018-12-12 at 9 49 39 am

screen shot 2018-12-12 at 9 46 28 am

### CODE REVIEW

**Comment by PhilWinchester** [comment]
Do we want to add an email to reach out to for this?

**Comment by PhilWinchester** [comment]
Also, do you mind posting the screenshots of what the pages look like?

File: docs/feeds.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+   * - Filename
+     - USERNAME-USD-yyyy-mm-dd.csv.zip
+
+
+Send via SFTP
+------------
+Please generate and provide us with a public RSA key to register as an authorized key for your users. Disable strict
+checking or remove previous entries from your SHH known_hosts file before each push; although the IP address
+remains the same, the host RSA key fingerprint will change.
```
Comment (csdev) [inline docs/feeds.rst]: This is not true -- the host key is set correctly and clients should not be instructed to ignore security warnings

**Comment by csdev** [comment]
We should also add a note about supported file types (csv, xml) and supported compressed file types (zip, gz)

**Pull request merged**